packages/lesswrong/components/posts/TruncatedAuthorsList.tsx

```diff
@@ -182,7 +183,7 @@ const TruncatedAuthorsList = ({post, expandContainer, classes}: {
 const TruncatedAuthorsListComponent = registerComponent(
   "TruncatedAuthorsList",
   TruncatedAuthorsList,
-  {styles},
+  {styles, stylePriority: -1},
```

Comment:
What's the reason for this one? (could you add a comment next to it if it's easy to explain too)